samples/bluetooth/central_uart/sample.yaml

```diff
@@ -10,6 +10,13 @@ tests:
       - nrf5340dk_nrf5340_cpuapp
       - nrf5340dk_nrf5340_cpuapp_ns
       - nrf21540dk_nrf52840
-    platform_allow: nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf52dk_nrf52810
+    platform_allow: nrf52dk_nrf52832 nrf52840dk_nrf52840
```

Comment:
Actually in the current state the sample was built for the `nrf52dk_nrf52810`, but it overflowed massively. In the change in sample.yaml the small target is actually built with different config as to allow the smaller footprint.
The sample does not appear in the documentation (and it didn't before) because it is in the `platform_allow` not in `integration_platforms`, I just kept it the same way with different conf file.
If having this board at all is a mistake than I will remove the entry completely.

Reply:
Let's remove it.

Reply:
Removed the nrf52810 target.